plugins/bif-solidity/profile.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "编译星火链网的 solidity 合约",
```

Comment:
could you update the description and the display name to be in english.
The description could include something like "Mandarin". thanks

Reply:
updated